### DISCUSSION TITLE
Missing CSS class after running build script

### DISCUSSION THREAD

**Discussion by InVinCiblezz** [tailwindlabs/tailwindcss#4908] [Help]
I used grid to display inputs. It works well when I using `npm start`.

![image](https://user-images.githubusercontent.com/18670530/124645456-1161a680-de59-11eb-9d56-d6566c967b1d.png)

However, if I used `npm run build` and use express or other services to target to build dict, all the grids I had set up were not working. 

![image](https://user-images.githubusercontent.com/18670530/124645599-3ce49100-de59-11eb-9102-907d15b05e2a.png)

also attach the tailwind setting
```js
// tailwind.config.js
module.exports = {
  purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {
      width: {
        '1/7': '14.2857143%',
        '2/7': '28.5714286%',
        '3/7': '42.8571429%',
        '4/7': '57.1428571%',
        '5/7': '71.4285714%',
        '6/7': '85.7142857%',
        '1/8': '12.5%',
        '2/8': '25%',
        '3/8': '37.5%',
        '4/8': '50%',
        '5/8': '62.5%',
        '6/8': '75%',
        '7/8': '87.5%',
      }
    },
  },
  variants: {
    extend: {},
  },
  plugins: [],
}
```

Thank you so much!

**Comment by InVinCiblezz** [comment]
I checked the elements, 
`@media (min-width: 1024px)
.lg\:col-span-4 {
    grid-column: span 4 / span 4;
}` is missing after npm run build.

**Comment by simonswiss** [comment]
Hey!

Without seeing code, I can only say this is most likely due to two things: 

1) a missing file path in your `purge` array
2) the use of a `dynamic` class like so:

```js
<div class={`lg:col-span-${props.columns}`}>
```

That won't work with Purge CSS! If your `lg:col-span-4` class is not being printed _in its entirety_ anywhere in the files you listed in your `purge` array, it will be removed from the CSS output.

If that's not one of these solutions, please share some code, because there is not much else troubleshooting possible without that.
